boot/cypress/MCUBootApp/image_ec256_mbedtls.c

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
This file could probably be refactored to go into `boot/bootutil/src/image_ec256.c`.